client/src/pages/Disclaimer/Disclaimer.tsx

```diff
@@ -0,0 +1,58 @@
+import NavigationBar from "../../components/NavigationBar";
+import { classnames } from "../../Utilities";
+import Footer from "../../components/Footer";
+
+export default function Disclaimer() {
+  return (
+    <>
+      <div className="w-1/2 mx-auto text-on-primary-container">
```

Comment:
Make this parent container take up the full height of the screen with the `h-screen` classname. Right now if the screen is taller than the content, there is blank space at the bottom
![image](https://user-images.githubusercontent.com/70448914/222586736-a70b48cf-71e4-402d-a45b-7700e0f2d70c.png)

Reply:
@BradleyLi is this resolved?